internal/api/connectors/cloudConnector.go

```diff
@@ -145,30 +138,26 @@ func (this *cloudConnectorClientImpl) SendCloudConnectorRequest(
 
 func (this *cloudConnectorClientImpl) GetConnectionStatus(
 	ctx context.Context,
-	account string,
 	orgID string,
 	recipient string,
 ) (status ConnectionStatus, err error) {
-	ctx = context.WithValue(ctx, accountKey, account)
 	ctx = context.WithValue(ctx, orgIDKey, orgID)
 
 	utils.GetLogFromContext(ctx).Debugw("Sending Cloud Connector status request",
-		"account", account,
+		"org_id", orgID,
 		"recipient", recipient,
 	)
 
-	res, err := this.client.V1ConnectionStatusMultiorgWithResponse(ctx, V1ConnectionStatusMultiorgJSONRequestBody{
-		Account: &account,
-		NodeId:  &recipient,
-	})
+	res, err := this.client.V2ConnectionStatusMultiorgWithResponse(ctx, ClientID(recipient))
 
 	if err != nil {
 		return "", err
 	}
 
-	if res.JSON200 == nil {
-		return "", unexpectedResponse(res.HTTPResponse)
+	connectionStatusResponse := ConnectionStatusResponse{}
```

Comment:
`ConnectionStatusResponse` is the return type of the v1 version of the operation. As we're using v2 of the operation, we should be working with `ConnectionStatusResponseV2`

Also, no need to unmarshal explicitly. You can instead get `ConnectionStatusResponseV2` by accessing res.JSON200.